src/Compilers/CSharp/Portable/Emitter/Model/MethodSymbolAdapter.cs

```diff
@@ -218,15 +218,13 @@ bool Cci.ISignature.ReturnValueIsByRef
         {
             get
             {
-                return this.ReturnType is ByRefReturnErrorTypeSymbol;
+                return this.RefKind == RefKind.Ref;
```

Comment:
How do we handle relative order of between custom modifiers and ByRef modifier in metadata?